docs/application-connector/docs/037-gs-register-secured-api.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Register a secured API
+type: Getting Started
+---
+
+The Metadata Service allows you to register a secured API for every service. The supported authentication methods are [Basic Authentication](https://tools.ietf.org/html/rfc7617) and [OAuth](https://tools.ietf.org/html/rfc6750).
+
+You can specify only one authentication method for every secured API you register. If you try to register and specify more than one authentication method, the Metadata Service returns a code `400` response.
```

Comment:
```suggestion
You can specify only one authentication method for every secured API you register. If you try to register and specify more than one authentication method, the Metadata Service returns the `400` status code.
```

Reply:
The service doesn't return a code, it returns a response which is identified by its code.

Reply:
a `400` code response then?